### ISSUE TITLE
Audio playing indicator in collapsed sidebar

### ISSUE THREAD

**Issue by infiniwave** [zen-browser/desktop#178]
### What feature would you like to see?

While the tab list is collapsed, add a way to show the tab where audio is playing.

### What platform are you seeing the problem on?

Windows

**Comment by DaitiDay** [comment]
I'm sharing here what I wrote in a closed issue as a possible solution to the problem.
I often use the audio icon to identify which tab is producing sound, and just as often, I mute a tab by pressing the audio icon without even opening the tab. Now the only options are 1) open the tab to pause the audio source or 2) right-click to mute the tab.
In Firefox 'vanilla,' I used an extension to get a vertical tab bar similar to the one Zen has by default, and in that case, the audio and mute icons would appear this way.
![Screenshot 2024-08-20 180704](https://github.com/user-attachments/assets/a7a8ac89-c530-46a1-82a2-1ebb2161937a)
It was enough to press the icon to toggle between audio and mute. Is a similar solution implementable in Zen?
Furthermore, I've noticed that the tab icon is miscentered when an audio source is present (the second tab in the image below is playing some music).
![image](https://github.com/user-attachments/assets/914fccea-7da7-462c-99f1-51cf779e96e0)
